catroid/src/org/catrobat/catroid/common/SoundInfo.java

```diff
@@ -76,23 +78,11 @@ public SoundInfo copySoundInfoForSprite(Sprite sprite) {
 
 	public String getAbsolutePath() {
 		if (fileName != null) {
-			return Utils.buildPath(getPathToSoundDirectory(), fileName);
-		} else {
-			return null;
-		}
-	}
-
-	public String getAbsolutePathBackPack() {
-		if (fileName != null) {
-			return Utils.buildPath(getPathToBackPackDirectory(), fileName);
-		} else {
-			return null;
-		}
-	}
-
-	public String getAbsolutePathBackPackSound() {
-		if (fileName != null) {
-			return Utils.buildPath(getPathToBackPackSoundDirectory(), fileName);
+			if (isBackpackSoundInfo) {
+				return Utils.buildPath(getPathToBackPackSoundDirectory(), fileName);
+			} else{
```

Comment:
@obusher @aried3r 
There is a jira ticket (https://jira.catrob.at/browse/CAT-1321) waiting to introduce all the neat checkstyle rules for those kind of "errors". But first, the AS formatter rules have to be configured (at least the 'Keep Maximum Blank Lines' and the array style) and also a whole formatter run is required (imo).